### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by marcelosousa**
This pull request makes a simple change to the README.md

### CODE REVIEW

**Comment by marcelosousa** [comment]
I can comment the pull request

**Comment by marcelosousa** [comment]
But I can't add a label to it

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # paddy
 
-This project aims to try out [reviewpad](https://reviewpad.com) :raised_hands:
\ No newline at end of file
+This project is used to play with the [Reviewpad Action](https://reviewpad.com) :raised_hands:
```
Comment (ferreiratiago) [inline README.md:3]: Good

**Review by ferreiratiago — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ferreiratiago) [reply README.md:3]: Good x2

**Review by ferreiratiago — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ferreiratiago) [inline README.md:3]: Try

**Review by ferreiratiago — CHANGES_REQUESTED** [review]

**Pull request closed without merging**